### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by korewa456**

### CODE REVIEW

**Review by cassidyjames — CHANGES_REQUESTED** [review]
Looks like spam or a typo?

**Review by AdNaN-rK — CHANGES_REQUESTED** [review]
Request changes

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ We're making changes to GitHub in a few phases, designed to cause as little disr
```
Comment (AdNaN-rK) [inline README.md:8]: Ch

**Review by AdNaN-rK — CHANGES_REQUESTED** [review]

**Comment by Bishem** [comment]
a ok

**Review by jaypalmandloi — APPROVED** [review]

**Review by ssbarnea — CHANGES_REQUESTED** [review]
Someone please close this PR

**Comment by henish1400** [comment]
ss

**Comment by ELHoussineT** [comment]
Close this.

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ We're making changes to GitHub in a few phases, designed to cause as little disr
 
 ## What's already changed
 
-- [We've updated GitHub.com](https://github.blog/changelog/2020-07-17-links-to-deleted-branches-now-redirect-to-the-default-branch/) to redirect links that contain the deleted branch name such as `master` to the corresponding link in the repository's new default branch.
+- [We've updated GitHub.com](https://github.blog/changelog/2020-07-17-links-to-deleted-branches-now-redirect-to-the-default-branch/) to redirect links that contain the deleted brancfenew default branch.
```
Comment (adarshank33) [inline README.md:11]: hiii

**Comment by KaspianDev** [comment]
:trollface:

**Pull request closed without merging**